config/groups.yaml

```
@@ -74,6 +74,12 @@
     - "2593423"
     - "2277898"
   id: 189602
+  name: "Low Expectations"
+  members:
+    - "2408602"
+    - "2192948"
+    - "2578334"
```

You're missing a dash here:

```suggestion
- name: "Low Expectations"
  members:
    - "2408602"
    - "2192948"
    - "2578334"
```